src/jobmaker/jobmaker.cfg

```diff
@@ -15,6 +15,9 @@ jobmaker = {
   # max gbt life cycle time seconds
   gbt_life_time = 60;
 
+  # max empty gbt life cycle time seconds
```

Comment:
Yes. The value would be better >= 10. If the non-empty job doesn't comes in 10 seconds,  the jobmaker will always produce a previous height job, that's unintended. 
So it is "empty_gbt_life_time" determines the life time of an empty job life time, if it's < 10, that will make the 10 sec job interval useless.
I think what's more should do next is making jobmaker to produce the job immediately at the non-empty gbt's arrival, if there is none, wait util the life time expired.